test/fw/ptl/utils/pbs_crayutils.py

```diff
@@ -82,6 +84,9 @@ def parse_apstat(self, options='-rn'):
                                   env=apstat_env)
         cmd_result = cmd_run['out']
         keys = cmd_result[0].split()
+        # Add a key Mode because State is composed of two list items
+        k2 = ['Mode']
+        keys = keys[0:3] + k2 + keys[3:len(keys)]
```

Comment:
You don't need to say len(keys).  You can leave off the end number of a splice and it means to the end of the list.  So in your case keys[3:]

Reply:
Done.